Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mhart/alpine-node:5.0
+
+ENV BUILD_PACKAGES bash curl-dev ruby-dev build-base libffi-dev libpq postgresql-dev git
+ENV RUBY_PACKAGES ruby ruby-io-console ruby-bundler ruby-raindrops ruby-nokogiri
+
+# Update and install all of the required packages.
+# At the end, remove the apk cache
+RUN apk update && \
+    apk upgrade && \
+    apk add $BUILD_PACKAGES && \
+    apk add $RUBY_PACKAGES && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir /app
+WORKDIR /app
+
+ADD . /app
+RUN bundle install --jobs 4 --without development test
```

Comment:
That's a good point! I was reading a bit on Dockerfile later and that's what I got out of it. The progrem I had with megatron (after copying `gemspec` file, too), was that it included more stuff as a reference (so things in `lib/` dir). I can try again with only adding `gemspec` and `lib` dir and see what happens.